src/material/schematics/ng-generate/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__-datasource.ts.template

```diff
@@ -103,6 +103,6 @@ export class <%= classify(name) %>DataSource extends DataSource<<%= classify(nam
 }
 
 /** Simple sort comparator for example ID/Name columns (for client-side sorting). */
-function compare(a, b, isAsc) {
+function compare(a: any, b: any, isAsc: boolean) {
```

Comment:
This could also be:
```ts
function compare(a: string | number, b: string | number, isAsc: boolean) {
```
But that might be slightly more work to customize. I'm happy to change it, but wanted to get some feedback on it first?

Reply:
Yeah, I'd prefer to avoid `any` in this situation